package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axelar-network/axelar-gmp-sdk-solidity",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
don't need to change this anymore. new release process is active now. update npm deps locally, run `npm run cs` to create a patch release changelog and push the changeset file to this PR. Then trigger the `Create Release PR` action to trigger the patch release

Reply:
got it. used `npm run cs` command for the changeset.